tests/save-images/test.sh

```diff
@@ -63,4 +63,27 @@ echo "=== ($LINENO): Test New Behaviour referencing remote via build ==="
 "$frontend" inspect myimage:buildtest >/dev/null
 "$frontend" inspect earthly-test-saveimage:test >/dev/null
 
+echo "=== ($LINENO): Test Disable Earthly Labels ==="
```

Comment:
Instead of having this test here, can you do it in an Earthfile instead?

Perhaps something along the lines of:

```
$ cat ./tests/with-docker-validate-labels/Earthfile 
VERSION --allow-disable-earthly-labels 0.8 0.8

ARG --global DIND_IMAGE=earthly/dind:alpine-3.19-docker-25.0.5-r0
FROM $DIND_IMAGE

all:
    BUILD +test-with-labels
    BUILD +test-without-labels

myimage-with-labels:
    FROM alpine
    COPY file /
    SAVE IMAGE myimage:test

myimage-without-labels:
    FROM alpine
    COPY file /
    SAVE IMAGE myimage:test

test-with-labels:
    WITH DOCKER --load=+myimage-with-labels
        RUN docker inspect myimage:test | jq -r '.[].Config.Labels' # ....
    END

test-without-labels:
    ...
```

then under `tests/Earthfile`, you can call it by adding it to

```
ga-no-qemu-group12:
    ...
    BUILD --pass-args ./with-docker-validate-labels+all
```